index.d.ts

```diff
@@ -250,45 +253,47 @@ declare module 'plaid' {
   }
 
   class Client {
-    constructor (
+    constructor(
       clientId: string,
       secret: string,
       publicKey: string,
       env: string,
-      options?: CoreOptions,
-    )
+      options?: CoreOptions
+    );
 
     exchangePublicToken(publicToken: string): Promise<TokenResponse>;
-    exchangePublicToken(publicToken: string,
-                        cb: Callback<TokenResponse>,
-    ): void;
+    exchangePublicToken(publicToken: string, cb: Callback<TokenResponse>): void;
```

Comment:
nit: seems inconsistent with the spacing for the other function declarations?
[e.g. here](https://github.com/plaid/plaid-node/pull/139/files#diff-b52768974e6bc0faccb7d4b75b162c99R269)

Reply:
I added some prettier settings that fixed it